node-notes-cli/app.js

```diff
@@ -0,0 +1,75 @@
+import * as fs from 'node:fs/promises';
+
+const fileContent = await fs.readFile('data.json', 'utf-8');
+const data = JSON.parse(fileContent);
+
+const fileWrite = async () => {
+  const write = await fs.writeFile('data.json', JSON.stringify(data, null, 2));
+  return write;
```

Comment:
You could combine lines 7 and 8:
`return await fs.writeFile.....`

And, actually, writeFile doesn't return anything useful, so you could just leave line 7 as:
`await fs.writeFile.....`